gen/dcos-config.yaml

```diff
@@ -254,6 +254,26 @@ package:
         },
         "max_configuration_attempts": {{ dcos_overlay_config_attempts }}
       }
+  - path: /etc/dcos/storage/csi/nfs.conf
```

Comment:
I don't think this is the proper location for this file. `/etc/dcos` doesn't currently exist on DC/OS nodes, does it?

I think putting it under `/opt/mesosphere/etc` makes more sense. How about `/opt/mesosphere/etc/storage/csi/nfs.conf`?

Reply:
We did the same for CNI network config file, see https://github.com/dcos/dcos/blob/2.1.0/gen/dcos-config.yaml#L202. Actually during installation, our installer will put that file under /opt/mesosphere/etc/storage/csi/ as you mentioned, I have verified this by installing the DC/OS package built from this PR.